packages/nuxt/src/core/features.ts

```diff
@@ -0,0 +1,35 @@
+import { addDependency } from 'nypm'
+import { isPackageExists } from 'local-pkg'
+import { logger } from '@nuxt/kit'
+import prompts from 'prompts'
+
+export async function ensurePackageInstalled (rootDir: string, name: string) {
+  if (isPackageExists(name)) {
+    return true
+  }
+
+  logger.info(`Package ${name} is missing`)
+
+  const { confirm } = await prompts({
+    type: 'confirm',
+    name: 'confirm',
+    message: `Do you want to install ${name} package?`,
+    initial: true
+  })
+
+  if (!confirm) {
+    return false
+  }
+
+  logger.info(`Installing ${name}...`)
+  try {
+    await addDependency(name, {
+      cwd: rootDir
```

Comment:
I think we should enable `dev: true` to add as dev dependency similar to `nuxt` one in starter